docs/examples/full_chain_odd.md

```diff
@@ -99,7 +100,8 @@ addSeeding(
 )
 ```
 
-The Combinatorial Kalman Filter (CKF) will use the seeds to propagate the trajectory forward and backward in time with the idea to find more measurements along the way. It combines (i.e. smoothes) these measurements and outputs reconstructed tracks which include smoothed track parameters for each measurement.
+The Combinatorial Kalman Filter (CKF) will use the seeds to propagate the trajectory forward and backward in time with the idea to find more measurements along the way.
+It combines (i.e. smooths) these measurements and outputs reconstructed tracks which include smoothed track parameters for each measurement.
```

Comment:
Is it really *smooths*?

Reply:
Should be:

https://www.oxfordlearnersdictionaries.com/definition/english/smooth_2